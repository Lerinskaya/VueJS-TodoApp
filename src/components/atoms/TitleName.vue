<template>
  <h1 :class="$style.todo__title">to do list</h1>
</template>

<script>
export default {};
</script>

<style lang="scss" module>
@import "@/styles/main.scss";
.todo__title {
  font-family: "Open Sans";
  background-color: $lightOrange;
  color: $brown;
  padding: 0.688rem 14.1rem;
  font-weight: 600;
  font-size: 1.25rem;
}
@media (max-width: 600px) {
  .todo__title {
    padding: 0.688rem 5rem;
  }
}
@media (max-width: 850px) {
  .todo__title {
    padding: 0.688rem 7rem;
  }
}
@media (max-width: 980px) {
  .todo__title {
    padding: 0.688rem 9.1rem;
    font-size: 1.2rem;
  }
}
@media (max-width: 450px) {
  .todo__title {
    padding: 0.688rem 2rem;
  }
}
</style>
