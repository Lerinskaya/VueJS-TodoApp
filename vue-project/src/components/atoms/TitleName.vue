<template>
  <h1 :class="$style.title">to do list</h1>
</template>

<script>
export default {};
</script>

<style lang="scss" module>
.title {
  font-family: "Open Sans";
  background-color: $lightOrange;
  color: $brown;
  padding: 0.688rem 14.1rem;
  font-weight: 600;
  font-size: 1.25rem;
}
@media (max-width: 600px) {
  .title {
    padding: 0.688rem 5rem;
  }
}
@media (max-width: 850px) {
  .title {
    padding: 0.688rem 7rem;
  }
}
@media (max-width: 980px) {
  .title {
    padding: 0.688rem 9.1rem;
    font-size: 1.2rem;
  }
}
@media (max-width: 450px) {
  .title {
    padding: 0.688rem 2rem;
  }
}
</style>
