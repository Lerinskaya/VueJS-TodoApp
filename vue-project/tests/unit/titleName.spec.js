import { mount } from '@vue/test-utils';
import TitleName from '@/components/atoms/TitleName.vue';

describe('TitleName.vue', () => {
    const wrapper = mount(TitleName);
    it('has name', () => {
        expect(wrapper.html()).toContain('<h1 class="title">to do list</h1>');
    });
});
