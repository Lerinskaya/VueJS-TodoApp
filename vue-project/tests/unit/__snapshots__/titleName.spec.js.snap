// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom TitleName Component Correct layout 1`] = `<h1 class="title">to do list</h1>`;

exports[`Atom TitleName Component has name 1`] = `<h1 class="title">to do list</h1>`;

exports[`TitleName.vue has name 1`] = `<h1 class="title">to do list</h1>`;
